.github/workflows/audit.yml

```diff
@@ -0,0 +1,18 @@
+name: Security audit
+
+on:
+  push:
+    paths: 
+      - '**/Cargo.toml'
+      - '**/Cargo.lock'
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  security_audit:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions-rs/audit-check@v1
```

Comment:
Shouldn't we generate lockfile before running check?

Reply:
`actions-rs/audit-check` will generate lockfile before checking.